contracts/src/v0.1/DataFeedRouter.sol

```diff
@@ -0,0 +1,242 @@
+     * time moves forward.
+     * @return answer is the answer for the given round
+     * @return startedAt is the timestamp when the round was started.
+     * (Only some AggregatorV3Interface implementations return meaningful values)
+     * @return updatedAt is the timestamp when the round last was updated (i.e.
+     * answer was last computed)
+     * @return answeredInRound is the round ID of the round in which the answer
+     * was computed.
+     * (Only some AggregatorV3Interface implementations return meaningful values)
+     * @dev Note that answer and updatedAt may change between queries.
+     */
+    function getRountData(
+        string calldata feedName,
+        uint80 roundId
+    )
+        external
+        view
+        validFeed(feedName)
+        returns (
+            uint80 id,
+            int256 answer,
+            uint256 startedAt,
+            uint256 updatedAt,
+            uint80 answeredInRound
+        )
+    {
+        return IAggregatorProxy(aggregatorProxies[feedName]).getRoundData(roundId);
+    }
+
+    /**
+     * @notice get data about the latest round. Consumers are encouraged to check
+     * that they're receiving fresh data by inspecting the updatedAt and
+     * answeredInRound return values.
+     * Note that different underlying implementations of AggregatorV3Interface
+     * have slightly different semantics for some of the return values. Consumers
+     * should determine what implementations they expect to receive
+     * data from and validate that they can properly handle return data from all
+     * of them.
+     * @param feedName the name of the datafeed (ex. btc-usdt)
+     * @return id is the round ID from the aggregator for which the data was
+     * retrieved combined with an phase to ensure that round IDs get larger as
+     * time moves forward.
+     * @return answer is the answer for the given round
+     * @return startedAt is the timestamp when the round was started.
+     * (Only some AggregatorV3Interface implementations return meaningful values)
+     * @return updatedAt is the timestamp when the round last was updated (i.e.
+     * answer was last computed)
+     * @return answeredInRound is the round ID of the round in which the answer
+     * was computed.
+     * (Only some AggregatorV3Interface implementations return meaningful values)
+     * @dev Note that answer and updatedAt may change between queries.
+     */
+    function latestRoundData(
+        string calldata feedName
+    )
+        external
+        view
+        validFeed(feedName)
+        returns (
+            uint80 id,
+            int256 answer,
+            uint256 startedAt,
+            uint256 updatedAt,
+            uint80 answeredInRound
+        )
+    {
+        return IAggregatorProxy(aggregatorProxies[feedName]).latestRoundData();
+    }
+
+    /**
+     * @notice Used if an aggregator contract has been proposed.
+     * @param feedName the name of the datafeed (ex. btc-usdt)
```

Comment:
`orakl-config` always uses names of feeds with uppercase letters. Let's use examples with uppercase in comments.

https://config.orakl.network/

```suggestion
     * @param feedName the name of the datafeed (ex. BTC-USDT)
```